website/layout/AutodocsLayout.js

```diff
@@ -451,10 +451,53 @@ var Autodocs = React.createClass({
               {metadata.next && <a className="docs-next" href={metadata.next + '.html#content'}>Next &rarr;</a>}
             </div>
           </div>
+
+          <EmbeddedSimulator shouldRender={metadata.runnable} metadata={metadata} />
+
         </section>
       </Site>
     );
   }
 });
 
+var EmbeddedSimulator = React.createClass({
+  render: function() {
+    if ( ! this.props.shouldRender ) return null;
```

Comment:
please always use {} and put the return null; on its own line.

Also, no space inside of () and after !